Dropdown/Dropdown.js

```diff
@@ -334,13 +343,13 @@ const DropdownBase = kind({
 		}
 	},
 
-	render: ({'aria-label': ariaLabel, ariaLabelledBy, children, css, direction, disabled, hasChildren, onClose, onOpen, onSelect, open, selected, skin, title, width, ...rest}) => {
+	render: ({adjusteddirection, 'aria-label': ariaLabel, ariaLabelledBy, children, css, direction, disabled, hasChildren, onAdjustDirection, onClose, onOpen, onSelect, open, selected, skin, title, width, ...rest}) => {
 		delete rest.rtl;
 
 		const ariaProps = extractAriaProps(rest);
 		const calcAriaProps = ariaLabel != null ? null : {role: 'region', 'aria-labelledby': ariaLabelledBy};
 
-		const popupProps = {'aria-live': null, children, direction, disabled, onSelect, open, selected, skin, skinVariants: skin === 'silicon' ? {'night': false} : {}, width, role: null};
+		const popupProps = {adjusteddirection, 'aria-live': null, children, direction, disabled, onSelect, open, selected, skin, skinVariants: skin === 'silicon' ? {'night': false} : {}, width, role: null};
```

Comment:
I think it would be better to update not to pass both `adjusteddirection` and `direction` prop. How about passing a `direction` prop with adjustedDirection value.

Reply:
I accepted your suggestion.